packages/server/tests/acceptance/estimateGasPrecompile.spec.ts

```diff
@@ -840,6 +1396,873 @@ describe('EstimatePrecompileContract tests', function () {
     isWithinDeviation(gasResult.gasUsed, estimateGasResponse, lowerPercentBound, upperPercentBound);
   });
 
+  //EGP-064
+  it('should call estimateGas with delete function with invalid token serial', async function () {
+    const invalidTokenSerial = '0x0000000000000000000000000000000000000AAA';
```

Comment:
Can you use an already existing constant from constants?

Reply:
Done